ui/js/folder.js

```diff
@@ -13,7 +13,7 @@
  * permissions and limitations under the License.
  */
 
-require(['jquery', 'underscore', 'oae.core'], function($, _, oae) {
+require(['jquery', 'underscore', 'oae.core', 'moment', 'async'], function($, _, oae, moment, async) {
```

Comment:
Do we actually use moment/async here?

Reply:
hehe I used to, but then refactored and moved code away. It's been removed now.